docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -780,7 +781,34 @@ directly or used in conditional statements.
 - `$file_size`
   : Size of the file for this row.
 
+### Replace tables
+
+The connector supports replacing an existing table, as
+an atomic operation. Atomic table replacement creates a new snapshot with the
+new table definition as part of the [table history](#delta-lake-history-table).
+
+To replace a table, use [`CREATE OR REPLACE TABLE`](/sql/create-table) or
+[`CREATE OR REPLACE AS`](/sql/create-table-as).
+
+In this example, a table `example_table` can be replaced by a completely new
+definition:
+
+```sql
+CREATE TABLE example_table (
+    a BIGINT,
+    b DATE,
+    c BIGINT)
+WITH (partitioning = "partitioned_by");
+```
+
+In this example, a new table is created with data:
+
+```sql
+CREATE OR REPLACE TABLE example_table
+WITH (partitioned_by = ARRAY['a'])
+AS SELECT * from another_table;
 (delta-lake-fte-support)=
+```
```

Comment:
This line should be before `(delta-lake-fte-support)=`

Reply:
yes.. might be github diff issue or actually wrong

Reply:
This is just weird.